AdyenActions/Components/Voucher/VoucherComponent.swift

```diff
@@ -13,7 +14,7 @@ internal protocol AnyVoucherActionHandler: ActionComponent {
 }
 
 /// A component that handles voucher action's.
-public final class VoucherComponent: AnyVoucherActionHandler {
+public final class VoucherComponent: AnyVoucherActionHandler, Cancellable {
```

Comment:
What is the point of `AnyVoucherActionHandler` ?
I don't see it is used..

Reply:
probably has no point any more, but its an internal protocol any way, we can remove it later, this is a bit out of scope